docs/fsharp/language-reference/byrefs.md

```diff
@@ -176,14 +176,19 @@ Although these rules very strongly restrict usage, they do so to fulfill the pro
 Byref returns from F# functions or members can be produced and consumed. When consuming a `byref`-returning method, the value is implicitly dereferenced. For example:
 
 ```fsharp
-let safeSum(bytes: Span<byte>) =
-    let mutable sum = 0
+let squareAndPrint (data : byref<int>) = 
+    let squared = data*data    // data is implicitly dereferenced
+    printfn "%d" squared
+```
```

Comment:
looks like this is failing the markdownlint check + build. If you add a newline here, then this should be mergeable.